firmware/src/sys/board.hpp

```diff
@@ -58,6 +58,9 @@ void resetWatchdog();
 
 void setStatusLed(bool state);
 void setCanLed(bool state);
+void DutycycleCounterPP();
+void DutycycleCounterMM();
+signed DutycycleCounterRead();
```

Comment:
Hmm so this method DutycycleCounterPP needs to be accessed from charger magnet and main. I tried putting it in magnet but then it was not accessible. 
What is the proper way to go about that?